src/Packages/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/Microsoft.CodeDom.Providers.DotNetCompilerPlatform.nuproj

```diff
@@ -29,9 +29,24 @@
         <NuGetContent Include="build\*">
             <Destination>build\net45</Destination>
         </NuGetContent>
+        <NuGetContent Include="build\*">
+            <Destination>build\net46</Destination>
+        </NuGetContent>
+        <NuGetContent Include="build\net45\*">
+            <Destination>build\net45</Destination>
+        </NuGetContent>
+        <NuGetContent Include="build\net46\*">
+            <Destination>build\net46</Destination>
+        </NuGetContent>
         <NuGetContent Include="tools\*.ps1">
             <Destination>tools\net45</Destination>
```

Comment:
any reason that this is the same as net45?

Reply:
anything under net45 folder will apply to project targeting 4.5+, if there is no other netxx folder in nupkg. The install.ps1 works in the same way no matter of the targeting framework.